src/Locale.php

```diff
@@ -230,13 +230,13 @@ private function isRequestIgnored(ServerRequestInterface $request): bool
     }
 
     /**
-     * @psalm-assert array<string, string> $this->supportedLocales
+     * @psalm-assert array<string, string> $supportedLocales
      *
      * @throws InvalidLocalesFormatException
      */
-    private function assertLocalesFormat(): void
+    private function validateSupportedLocales(array $supportedLocales): void
```

Comment:
When the validation function throws an exception it is usually called `assert*()`.